src/Adapter/Module/ModuleDataProvider.php

```diff
@@ -114,6 +126,7 @@ public function isModuleMainClassValid($name)
         $content = preg_replace('/\n[\s\t]*?use\s.*?;/', '', $file);
         $content = preg_replace('/\n[\s\t]*?namespace\s.*?;/', '', $content);
```

Comment:
> eval is evil. Mickaël Andrieu - now

Can you give a try to Expression Language Component ? http://symfony.com/doc/current/components/expression_language/introduction.html#usage